packages/api-explorer/src/components/SideNav/SideNavMethods.spec.tsx

```diff
@@ -52,6 +52,29 @@ describe('SideNavMethods', () => {
     )
   })
 
+  test('method opens successfully', () => {
+    renderWithRouterAndReduxProvider(
+      <SideNavMethods methods={methods} tag={tag} specKey={'3.1'} />
+    )
+    const initExpandedPanels = screen.queryByRole('region')
+    expect(initExpandedPanels).not.toBeInTheDocument()
+    userEvent.click(screen.getByText(tag))
+    const expandedPanelsAfterClick = screen.getByRole('region')
+    expect(expandedPanelsAfterClick).toBeInTheDocument()
+  })
+
+  test('method closes successfully', () => {
+    renderWithRouterAndReduxProvider(
+      <SideNavMethods methods={methods} tag={tag} specKey={'3.1'} />
+    )
+    userEvent.click(screen.getByText(tag))
```

Comment:
in addition to doing the refactor as mentioned above, use `defaultOpen` prop to have this be opened on render instead of clicking it. It reduces set up time (we have already tested opening in another test) and makes the test more concise